webpack/webpack.config.prod.js

```diff
@@ -15,15 +16,19 @@ config.plugins.push(
         'process.env.QUANDL_API_KEY': JSON.stringify(process.env.QUANDL_API_KEY),
         'process.env.QUANDL_KEY': JSON.stringify(process.env.QUANDL_KEY)
     }),
-    new webpack.optimize.UglifyJsPlugin({
-        compressor: {
-            warnings: false
-        }
-    }),
     new CopyWebpackPlugin([
         { from: 'src/static' },
         { from: 'openfin-config/app.prod.json', to: './app.json' }
     ])
 );
+config.optimization = {
+    minimizer: [
+        new UglifyJsPlugin({
```

Comment:
I thought webpack 4 in `mode: production` automatically adds in uglification.

Reply:
In fact I don't see anywhere where `mode` is being specified -- does this come with a warning?

Reply:
This is the npm build command `npm run clean && webpack -p --config webpack/webpack.config.prod.js` where production mode is applied.
Agree, let's use webpack defaults for now.

Reply:
I mean this: https://webpack.js.org/concepts/mode/ I don't see that anywhere, and it should be specified (it's what turns on the minification etc by default)

Reply:
Documentation says 
> Possible values for mode are: none, development or production(default).
```
Usage 
Just provide the mode option in the config:
module.exports = {
  mode: 'production'
};
or pass it as a CLI argument:
webpack --mode=production
```
The latter is what we do in the build command
```
npm run clean && webpack -p --config webpack/webpack.config.prod.js
```